src/NearOrg/Notifications/NotificationButton.jsx

```diff
@@ -15,6 +15,11 @@ State.init({
   open: false,
 });
 
+const handleDropdownOpen = () => {
+  if (notificationsCount === 0) return;
```

Comment:
As far as notification counter is working fine maybe it's good to hide preview dropdown is there are no new notifications. If you don't like this approach I'll remove it

cc @marcinbodnar @charleslavon

Reply:
That's a good question @shelegdmitriy. @ravigautham should we change the UX such that if you have no unread notifications when you click the alert icon we show an empty box or some other affordance?

@shelegdmitriy I don't think we'll want to take no action upon clicking, because that might lead one to think that the UI is broken. Let's remove this for now and handle this potential UX improvement separately.

Reply:
Ok, I'll remove this. Let me explain more how I understand this approach (and how it's implemented here).

When user has some notifications - everything works as before. But when there are no any we are not showing dropdown menu but you can still click on bell button and see all the notifications on it's own page (like in our old implementation).

This is just my proposal. Totally agree that it could be improved in future and it's out of scope of this task.

Reply:
Ah, I understand it much better now. Thanks @shelegdmitriy. That sounds like a worthwhile improvement to include on another PR.